accord-core/src/main/java/accord/impl/InMemoryCommandStore.java

```diff
@@ -629,6 +629,8 @@ public <T> T mapReduce(Seekables<?, ?> keysOrRanges, Ranges slice, TestKind test
             Map<Range, List<Command>> collect = new TreeMap<>(Range::compare);
             commandStore.rangeCommands.forEach(((txnId, rangeCommand) -> {
                 Command command = rangeCommand.command.value();
+                if (command == null)
```

Comment:
I don't think this should ever be null, so we might want to look more closely into why this happens?

Reply:
switched this to a nonNull check